kornia/augmentation/_2d/geometric/affine.py

```diff
@@ -0,0 +1,131 @@
+from typing import Dict, Optional, Tuple, Union, cast
+
+import torch
+
+from kornia.augmentation import random_generator as rg
+from kornia.augmentation._2d.geometric.base import GeometricAugmentationBase2D
```

Comment:
I think that would be interesting to expose `GeometricAugmentationBase2D` to enable kornia users to easiliy create their custom stuff 

```python
from kornia.augmentation import GeometricAugmentationBase2D
```

Reply:
Yes, I already exposed that